controllers/save_parameter_settings.js

```diff
@@ -1,46 +1,52 @@
 module.exports = function SaveParameterSettingsControllerModule(pb) {
   var util = pb.util;
 
-  function SaveParameterSettings() {}
+  function SaveParameterSettings(options) {
+    if (options){
+      this.site = options.site ? options.site : '';
+    }else {
+      this.site = '';
+    }
+    this.siteQueryService = new pb.SiteQueryService(this.site, true);
+  }
```

Comment:
This is a controller.  We don't need to pass in options to the constructor.  options is for services only.  this.site and this.siteqQueryService are already defined in BaseController.